### PR TITLE
test: add coverage scenario for ostree

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Comment by jelly** [comment]
Looks good! https://cockpit-logs.us-east-1.linodeobjects.com/pull-397-20230727-174152-0346255e-fedora-coreos-devel/Coverage/index.html

File: Makefile
Comment (martinpitt) [inline Makefile:108]: Please revert the "webpack" word, "bundler" is correct. This isn't using webpack.

**Comment by martinpitt** [comment]
Nice! do we have any chance to exclude pkg/lib/ from the coverage report? *by design* we aren't going to use all shared components, and whenever we add a new one, the coverage would appear to go down.

**Comment by martinpitt** [comment]
Once this lands, @jelly can you please file a bots PR about enabling the /devel scenario for all PRs?

**Comment by jelly** [comment]
> Nice! do we have any chance to exclude pkg/lib/ from the coverage report? _by design_ we aren't going to use all shared components, and whenever we add a new one, the coverage would appear to go down.

We can, but iirc Marius liked including `pkg/lib` and we do so in machines/podman so I think we need to make a more general team decision :) (For stats it does make sense hmm to exclude it)

**Review by martinpitt — APPROVED** [review]
Yeah, it doesn't make sense to report coverage percentage for foreign code. We don't do this for anything in node_modules/ either, for example. Same thing for podman.

That said, this doesn't have to block this PR, thanks!

**Pull request merged**
